public/assets/.sprockets-manifest-ac0698f63e53e9970afe149f222e4521.json

```diff
@@ -0,0 +1 @@
+{"files":{"application-94e358a5b1ab7e071e922342be023d099f179d8a541803c3387e59c8d386a8b3.js":{"logical_path":"application.js","mtime":"2020-05-11T19:18:36-06:00","size":23149,"digest":"94e358a5b1ab7e071e922342be023d099f179d8a541803c3387e59c8d386a8b3","integrity":"sha256-lONYpbGrfgcekiNCvgI9CZ8XnYpUGAPDOH5ZyNOGqLM="},"application-f0b77d1b81730b2a62e3aa66ea37a5b18d6969e24b9e117b1468ce6923051ee8.css":{"logical_path":"application.css","mtime":"2020-05-12T19:45:51-06:00","size":989,"digest":"f0b77d1b81730b2a62e3aa66ea37a5b18d6969e24b9e117b1468ce6923051ee8","integrity":"sha256-8Ld9G4FzCypi46pm6jelsY1paeJLnhF7FGjOaSMFHug="}},"assets":{"application.js":"application-94e358a5b1ab7e071e922342be023d099f179d8a541803c3387e59c8d386a8b3.js","application.css":"application-f0b77d1b81730b2a62e3aa66ea37a5b18d6969e24b9e117b1468ce6923051ee8.css"}}
```

Comment:
I honestly have no idea what this is. I was trying to fix something in a test and I tried a random command I found on google and this happened!

Reply:
I'd like to delete it but I'm a little scared!

Reply:
Yeah, let's keep it until we figure out what it means.